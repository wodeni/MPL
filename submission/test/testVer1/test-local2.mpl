int foo()
{	
    int c;
    int e;
    int w;
    int n;
    int s;
    int ne;
    int se;
    int nw;
    int sw;
    c = #C;
    e = #E;
    w = #W;
    n = #N;
    s = #S;
    ne = #NE;
    se = #SE;
    nw = #NW;
    sw = #SW;
    
    if (c==5) 
	print(c);	
    if (c==5) 
	print(e);	
    if (c==5) 
	print(w);	
    if (c==5) 
	print(n);	
    if (c==5) 
	print(s);	
    if (c==5) 
	print(ne);	
    if (c==5) 
	print(se);	
    if (c==5) 
	print(nw);	
    if (c==5) 
	print(sw);	
    return c;
}

int main()
{
    Mat<int>[3][3] m;
    m = [1,2,3;4,5,6;7,8,9];
    printm(m);
    foo @ m;
    return 0;
}
