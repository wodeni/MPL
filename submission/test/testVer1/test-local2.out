[
1, 2, 3;
4, 5, 6;
7, 8, 9;
]
5
6
4
2
8
3
9
1
7
